packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -42,54 +49,44 @@ const TimeDuration = styled.time`
   margin: 0 ${GEL_SPACING_HLF};
 `;
 
-const IndexAlsosMediaIndicator = styled.span`
-  & > svg {
-    margin: 0;
-  }
-`;
-
 const MediaIndicator = ({
   datetime,
   duration,
   type,
   topStory,
   service,
-  indexAlsos,
-}) =>
-  indexAlsos ? (
-    <IndexAlsosMediaIndicator aria-hidden="true">
+  isInline,
+}) => (
+  <MediaIndicatorWrapper
+    aria-hidden="true"
+    topStory={topStory}
+    service={service}
+    isInline={isInline}
+  >
+    <FlexWrapper>
       {mediaIcons[type]}
-    </IndexAlsosMediaIndicator>
-  ) : (
-    <MediaIndicatorWrapper
-      aria-hidden="true"
-      topStory={topStory}
-      service={service}
-    >
-      <FlexWrapper>
-        {mediaIcons[type]}
-        {duration && datetime && (
-          <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
-        )}
-      </FlexWrapper>
-    </MediaIndicatorWrapper>
-  );
+      {duration && datetime && (
+        <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
```

Comment:
Just had a UX review of the Desktop layout, and the UX designers said that when we use the media indicator inline, the duration shouldn't be displayed.  
There are two ways to do this. One is to couple that logic here in Psammead - so when you have `isInline`, you don't render the duration component.  The other way is to handle this where this component is being used.  I think this depends on whether we will *ever* have the duration being displayed when in the 'inline' view. I'll just double check with UX about this.

Reply:
I'm happy to include this in this PR, but will wait to make sure

Reply:
@jblaut Thanks. Confirmed with UX that for now, it can be coupled to the `isInline` prop since that is the general usecase.